prov/efa/src/efa_prov_info.c

```diff
@@ -254,12 +220,26 @@ void efa_prov_info_set_tx_rx_attr(struct fi_info *prov_info,
 	if (ep_type == FI_EP_RDM) {
 		*prov_info->tx_attr	= efa_rdm_tx_attr;
 		*prov_info->rx_attr	= efa_rdm_rx_attr;
+
+		prov_info->tx_attr->op_flags |= EFA_RDM_TX_OP_FLAGS;
+		prov_info->rx_attr->op_flags |= FI_COMPLETION;
+
+		/* If the device supports FI_RMA, also set tx_attr and rx_attr */
+		if (prov_info->caps & FI_RMA) {
+			prov_info->tx_attr->caps |= OFI_TX_RMA_CAPS;
+			prov_info->rx_attr->caps |= OFI_RX_RMA_CAPS;
+		}
+
+		/* efa-direct path requires FI_CONTEXT2 */
+		prov_info->tx_attr->mode |= FI_CONTEXT2;
```

Comment:
Both dgram and direct requires FI_CONTEXT2.

Reply:
fixed